test/JDBC/expected/non_default_server_collation/chinese_prc_ci_as/BABEL-CASE_EXPR-vu-verify.out

```diff
@@ -0,0 +1,5368 @@
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN N'بطاقة التسجيل - قياسية'
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    '
+END AS RESULT
+GO
+~~START~~
+nvarchar
+abcd
+~~END~~
+
+
+SELECT CASE N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN 'abcd'
+   WHEN N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS THEN CAST(N'  比尔·拉莫斯    ' AS VARCHAR(100))
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN N'بطاقة التسجيل - قياسية'
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    '
+END AS RESULT
+GO
+~~START~~
+nvarchar
+  比尔·拉莫斯    
+~~END~~
+
+
+SELECT CASE N'بطاقة التسجيل - قياسية' COLLATE CHINESE_PRC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN 'abcd'
+   WHEN N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS THEN N'  比尔·拉莫斯    '
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN CAST(N'بطاقة التسجيل - قياسية' AS VARCHAR(100))
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    '
+END AS RESULT
+GO
+~~START~~
+nvarchar
+بطاقة التسجيل - قياسية
+~~END~~
+
+
+SELECT CASE N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN 'abcd'
+   WHEN N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS THEN N'  比尔·拉莫斯    '
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN N'بطاقة التسجيل - قياسية'
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN CAST(N'  abc🙂defghi🙂🙂    ' AS VARCHAR(100))
+END AS RESULT
+GO
+~~START~~
+nvarchar
+  abc🙂defghi🙂🙂    
+~~END~~
+
+
+SELECT CASE 'abcd' COLLATE ARABIC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN CAST('abcd' AS VARCHAR(100))
+   WHEN N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS THEN N'  比尔·拉莫斯    '
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN N'بطاقة التسجيل - قياسية'
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    '
+END AS RESULT
+GO
+~~START~~
+nvarchar
+abcd
+~~END~~
+
+
+SELECT CASE N'  比尔·拉莫斯    ' COLLATE ARABIC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN 'abcd'
+   WHEN N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS THEN CAST(N'  比尔·拉莫斯    ' AS VARCHAR(100))
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN N'بطاقة التسجيل - قياسية'
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    '
+END AS RESULT
+GO
+~~START~~
+nvarchar
+  比尔·拉莫斯    
+~~END~~
+
+
+SELECT CASE N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN 'abcd'
+   WHEN N'  比尔·拉莫斯    ' COLLATE CHINESE_PRC_CI_AS THEN N'  比尔·拉莫斯    '
+   WHEN N'بطاقة التسجيل - قياسية' COLLATE ARABIC_CI_AS THEN CAST(N'بطاقة التسجيل - قياسية' AS VARCHAR(100))
+   WHEN N'  abc🙂defghi🙂🙂    ' COLLATE CHINESE_PRC_CI_AS THEN N'  abc🙂defghi🙂🙂    '
+END AS RESULT
+GO
+~~START~~
+nvarchar
+بطاقة التسجيل - قياسية
+~~END~~
+
+
+SELECT CASE N'  abc🙂defghi🙂🙂    ' COLLATE ARABIC_CI_AS
+   WHEN 'abcd' COLLATE japanese_ci_as THEN 'abcd'
```

Comment:
LGTM but we had an ask to put COLLATE as the resultant node as well

Reply:
Added those tests.